### PR TITLE
Add .yaml config for `QuiltVQA`

### LINKED ISSUE #908: Add `QuiltVQA` dataset
https://huggingface.co/datasets/wisdomik/Quilt_VQA

### PR DESCRIPTION
**Pull request by nkaenzig**
Closes #908

Building up on https://github.com/kaiko-ai/eva/pull/912 and https://github.com/kaiko-ai/eva/pull/914, this PR adds a a fully functional evaluation config for `QuiltVQA`. This is the first free-form visual question answering task in `eva` - using a G-Eval LLM Judge as metric.

## How to test?

```bash
export GEMINI_API_KEY=<your-secret>
DATA_ROOT=/path/to/download/dataset/to DOWNLOAD_DATA=true MODEL_NAME=google/gemini-2.5-flash-lite eva test --config configs/multimodal/pathology/online/free_form/quilt_vqa.yaml
```

image

### CODE REVIEW

File: src/eva/language/metrics/llm_judge/base.py [hunk 1]
```diff
@@ -15,7 +17,7 @@ class LLMJudge(Generic[JudgeOutput], abc.ABC):
 
     def __init__(
         self,
-        model: wrappers.LanguageModel,
+        model: wrappers.LanguageModel | nn.Module,
```
Comment (MaxFeucht) [inline src/eva/language/metrics/llm_judge/base.py]: Which `nn.Module` are we expecting here?

File: configs/multimodal/pathology/offline/free_form/quilt_vqa.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          model:
+            class_path: eva.multimodal.models.wrappers.ModelFromRegistry
+            init_args:
+              model_name: ${oc.env:MODEL_NAME, anthropic/claude-3-7-sonnet-20250219}
+              model_extra_kwargs: ${oc.env:MODEL_EXTRA_KWARGS, null}
+          overwrite: false
+model:
+  class_path: eva.language.models.OfflineLanguageModule
+  init_args:
+    metrics:
```
[the remark this answers is not in the document]
Comment (nkaenzig) [reply configs/multimodal/pathology/offline/free_form/quilt_vqa.yaml]: It is also possible to split this into two different .yaml configs, one only containing the `predict` dataset, and the other one the `test` dataset and the metrics. There are definitively usecases where you want to more strictly separate the two, especially if inference is done outside `eva`. However, when we built the `vision` module be put quite a bit of thought into this and decided to support this single config format, as this also enables multi-step commands like `predict_fit` with one single config path in the CLI call, which can be handy.

File: src/eva/language/metrics/llm_judge/g_eval/judge.py [hunk 3]
```diff
@@ -77,13 +79,21 @@ def evaluate(self, batch: PredictionBatch[List[str]]) -> List[int | None]:
...
+            )
+        )
 
         results = list(map(self._parse_output, outputs["generated_text"]))
         scores, reasons = zip(*results, strict=False)
 
         return list(scores)
 
-    def _load_model(self, model: wrappers.LanguageModel | str | None) -> wrappers.LanguageModel:
+    def _load_model(self, model: wrappers.LanguageModel | nn.Module | str | None) -> nn.Module:
```
[the remark this answers is not in the document]
Comment (nkaenzig) [reply src/eva/language/metrics/llm_judge/g_eval/judge.py]: https://github.com/kaiko-ai/eva/pull/919#discussion_r2450641804
